apps/cowswap-frontend/src/modules/hooksStore/pure/HookDetailHeader/index.tsx

```diff
@@ -1,25 +1,43 @@
+import { HookDappWalletCompatibility } from '@cowprotocol/hook-dapp-lib'
+
 import * as styled from './styled'
 
 import { HookDapp } from '../../types/hooks'
 
 interface HookDetailHeaderProps {
   dapp: HookDapp
+  walletType?: HookDappWalletCompatibility
   onSelect?: () => void
   iconSize?: number
   gap?: number
   padding?: string
 }
 
-export function HookDetailHeader({ dapp, onSelect, iconSize, gap, padding }: HookDetailHeaderProps) {
+export function HookDetailHeader({ dapp, walletType, onSelect, iconSize, gap, padding }: HookDetailHeaderProps) {
   const { name, image, descriptionShort } = dapp
 
+  const isCompatible =
```

Comment:
The code is duplicated in `HookListItem` component, I suggest to move it in a util function